app/views/worker/edit.phtml

```diff
@@ -0,0 +1,78 @@
+<h3>Edit worker</h3>
+
+{{ form('worker/edit', 'method': 'post') }}
+<fieldset>
+
+    <div class='form-group'>
+        <label for='id'>Select worker</label>
+        <div class='row justify-content-center'>
+            <div class='col-xs-12 col-sm-6 col-md-4 col-lg-3'>
+                <div class='controls'>
+                    {{ select(
+                    'id',
+                    workers,
+                    'using': ['id', 'fullName'],
+                    'required': 'true',
+                    'class': 'form-control',
+                    'useEmpty': 'true',
+                    'emptyText' : 'No worker is selected',
+                    'emptyValue' : ''
+                    )}}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class='form-group'>
+        <label for='lastName'>Last name</label>
+        <div class='row justify-content-center'>
+            <div class='col-xs-12 col-sm-6 col-md-4 col-lg-3'>
+                {% if errors is defined and errors.filter('lastName')|length > 0 %}
+                <div class='text-danger'>
+                    {{ errors.filter('lastName')[0].getMessage() }}
```

Comment:
you should never access array by keys manually
eg: $array[0]... loop over them or find other solutions